### DISCUSSION TITLE
kodi

### DISCUSSION THREAD

**Discussion by andrevj1** [community/community#49297] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i am looking for a   repository  for DSTV now for kodi   the link  that i had is broken it was slyguy.uk
if anyone know of anything   repository  that i can used for DSTV NOW let me know please

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @andrevj1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by andrevj1** [reply]
Thanx when will i get a responce to where can i download a zip file that work on kodi  for dstv now because i try every other zip file
